packages/wrangler/src/cfetch/index.ts

```diff
@@ -44,6 +45,43 @@ export async function fetchResult<ResponseType>(
 	}
 }
 
+/**
+ * Make a fetch request, and extract the `result` from the JSON response.
+ */
```

Comment:
Can you update this doc comment to more accurately reflect what this function actually does?